src/containers/App/App.test.js

```diff
@@ -1,106 +1,718 @@
+      }],
+      noiEligible: [{
+        id: "2",
+        type: "job",
+        attributes: {
+          job_type: 'Labor',
+          job_site_name: 'Home',
+          job_site_contact_name: 'Taryn',
+          job_site_address: '200 Washington St.', job_site_address_line_2: '', job_site_city: 'Denver',
+          job_site_state: 'CO', job_site_zip_code: '80201', completion_date: "2020-10-01T04:05:06.000Z",
+          material_cost: 200,
+          labor_cost: 200,
+          total_cost: 400,
+          description_of_work: 'blah',
+          client_company_name: 'Microsoft',
+          business_address: '12 Tree Ave',
+          business_address_line_2: 'Suite 200',
+          business_city: 'Seattle',
+          business_state: 'WA', business_zip_code: '99900', additional_info: 'Amazon is cool',
+          status: 'NOI Eligible'
+        }
+      }],
+      lienEligible: [],
+      releaseEligible: []
+    }
+
+    const mockInfo = {
+        attributes: {
+          job_type: 'Labor',
+          job_site_name: 'Home',
+          job_site_contact_name: 'Taryn',
+          job_site_address: '200 Washington St.', job_site_address_line_2: '', job_site_city: 'Denver',
+          job_site_state: 'CO', job_site_zip_code: '80201', completion_date: currentDate,
+          material_cost: 200,
+          labor_cost: 200,
+          total_cost: 400,
+          description_of_work: 'blah',
+          client_company_name: 'Amazon',
+          business_address: '12 Tree Ave',
+          business_address_line_2: 'Suite 200',
+          business_city: 'Seattle',
+          business_state: 'WA', business_zip_code: '99900', additional_info: 'Amazon is cool',
+          status: 'good standing'
+        }
+    }
+
+    getAllJobs.mockResolvedValueOnce(allJobs)
+
+    const store = mockStore({
+      allJobs: allJobs,
+      jobInfo: mockInfo
+    })
+
+    render(
+      <Provider store={store}>
+        <MemoryRouter>
+          <App />
+        </MemoryRouter>
+      </Provider>
+    )
+
+    const header = await waitFor(() => screen.getByRole('heading', { name: 'What do you want to do?'}))
+    const eligibleJobsButton = screen.getByRole('button', { name: 'NOI Eligible Jobs' })
+
+    fireEvent.click(eligibleJobsButton);
+
+    const card = await waitFor(() => screen.getByText('Amazon'));
+
+    expect(card).toBeInTheDocument()
+
+    fireEvent.click(card);
+
+    const info = await waitFor(() => screen.getByText('Amazon is cool'));
+
+    expect(info).toBeInTheDocument()
+
+  })
+  //maybe do these in job details since these are specific to that container?
+  // it('should allow a user to remove a job on click in job details', () => {
+  //   // post request
+  //   // redirects to homepage
+  //
+  // })
+  // it('should allow a user to submit an noi on click', () => {
+  //
+  // })
+  // it('should allow a user to submit a lien on click', () => {
+  //
+  // })
+  // it('should allow a user to release a lien on click', () => {
+  //
+  // })
+})
```

Comment:
I've got tests in the UserMessages that check if `handleClick` on each of these buttons was invoked. Will probably need to add another test checking if the actual patch request is invoked, but I'm struggling to get those fetch tests setup correctly.